packages/terra-form-select/src/terra-dev-site/test/form-select/SelectVariantsWith500Options.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
```

Comment:
This file is just to verify the changes and will be removed before merging them.

Reply:
we might need to keep this file to test the PR code changes with WDIO.

Reply:
I have removed this test. Changes can be verified in this test https://github.com/cerner/terra-core/pull/3464/files#diff-5285632fbb1d1102a5455105fc54dbfe0e7adf24e64a59f1eeb38ea2cdfd4736R8